dts/bindings/sensor/bosch,bmp085.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2024, Mario Paja
+# SPDX-License-Identifier: Apache-2.0
+
+description: Bosch BMP085 pressure sensor
+
+compatible: "bosch,bmp085"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  oversampling:
+    type: int
+    default: 0
```

Comment:
default value needs justification in the description https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html

Reply:
done :)

Reply:
where did the default value go? :)
You also introduced a compliance error, it would seem

Reply:
@kartben 
I did not find any particular reason why  the default value it should be 0 but from the data sheet is seems that it is required for the pressure measurement. For this reason I made it required :)
I will check it immediately :)

Reply:
@kartben @decsny 

Would this explanation be conform the rules? Although it doesn't seem like there is any specified default value from the data sheet. 

```
    default: 1
    description: |
      The oversampling rate. Required for the pressure measurement.
      The default corresponds to the standard oversampling rate.
```